Engine/ac/draw.cpp

```diff
@@ -914,14 +914,15 @@ void sort_out_char_sprite_walk_behind(int actspsIndex, int xx, int yy, int basel
 void clear_draw_list() {
     thingsToDrawList.clear();
 }
-void add_thing_to_draw(IDriverDependantBitmap* bmp, int x, int y, int trans, bool alphaChannel) {
+void add_thing_to_draw(IDriverDependantBitmap* bmp, int x, int y, int trans, bool alphaChannel, int blendMode = 0) {
```

Comment:
Please use actual blend constant instead of "0", since blend mode is enum and supposed to stay in defined range.